README.md

```diff
@@ -61,6 +61,20 @@ If you happened to use `git clone` only, you won't see the `cartographer` folder
 
 `git submodule update --init`
 
+#### (optional) Using Canon Images
+
+Once downloaded, you can enter the repo and create a docker container for building either an `arm64` or `amd64` binaries  using Viam's canon tool by running one of the following commands:
```

Comment:
should describe how to download canon here

Reply:
+1, and same as https://github.com/viamrobotics/viam-orb-slam3/pull/25#discussion_r1148029324